module/zcommon/zpool_prop.c

```diff
@@ -50,6 +50,14 @@ zpool_prop_get_table(void)
 void
 zpool_prop_init(void)
 {
+	struct zfs_mod_supported_features *sfeatures =
+	    zfs_mod_list_supported(ZFS_SYSFS_POOL_PROPERTIES);
+#define	zprop_register_impl(...) zprop_register_impl(__VA_ARGS__, sfeatures)
```

Comment:
The use of these #define reduced the amount of code changes but I'm not sure it's OK in this code base. 

@behlendorf Any concern here? Guess I'd like to avoid someone needing to do a cleanup in the future.

Reply:
Normally we've updated all of the callers, offhand I can't think of anywhere we've done this.  My only concern would be that this makes things a tiny bit harder to read, though I don't think it's a big deal either way.  Personally I'd opt for updating the callers.   If you'd prefer to stick with the `#define` that's fine, but I would recommend we then `#undef` them at the end of the functions to limit their scope.